test/bridge/testase.py

```diff
@@ -0,0 +1,360 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Tests for the cclib2ase bridge in cclib."""
+
+import unittest
+
+import numpy as np
+
+from cclib import ccopen
+from cclib.bridge import cclib2ase
+
+from ase import Atoms
+from ase.optimize import GPMin, QuasiNewton
+from ase.calculators.emt import EMT
+from ase.calculators.psi4 import Psi4
+
+
+class ASETest(unittest.TestCase):
+    """Tests for the cclib2ase bridge in cclib."""
+
+    def test_makease_allows_optimization(self):
+        """Ensure makease works from direct input."""
+        h2 = cclib2ase.makease([[0, 0, 0], [0, 0, 0.7]], [1, 1])
+
+        # check whether converting back gives the expected data
+        data = cclib2ase.makecclib(h2)
+        assert np.allclose(data.atomcoords, [[[0, 0, 0], [0, 0, 0.7]]])
+        assert np.allclose(data.atomnos, [1, 1])
+        assert np.allclose(data.atommasses, [1.008, 1.008])
+        assert np.isclose(data.natom, 2)
+        assert np.isclose(data.charge, 0)
+        assert np.isclose(data.mult, 1)
+        assert np.isclose(data.temperature, 0)
+
+        # front page example of ASE (https://wiki.fysik.dtu.dk/ase/)
+        h2.calc = Psi4(
+            label="h2", method="PBE", basis="3-21G", num_threads="max"
+        )
+        opt = GPMin(h2)  # optimize geometry
```

Comment:
Why do we need to optimize a geometry to test code that converts data structures?

Reply:
No need actually, I just wanted to ensure the returned object would work as expected. Should we remove it? I could make tests calculate energies and forces only.

Reply:
> Why do we need to optimize a geometry to test code that converts data structures?

Removed all optimizations.